dockerfile

```diff
@@ -0,0 +1,26 @@
+# Use a Node.js base image
+FROM node:slim
+
+# Set the working directory in the container
+WORKDIR /cdda-guide
+
+# Install git
+RUN apt-get update && apt-get install -y git
+
+# Clone the specific repository
+RUN git clone https://github.com/DoiiarX/cdda-guide /cdda-guide

```

Comment:
If you want to contribute a Dockerfile, it would make sense to have it use the main `nornagon` repo rather than your fork

I would also point out as a warning to others that DoiiarX's fork adds Microsoft and Google analytics